tests/kernel/install_ltp.pm

```diff
@@ -346,7 +346,7 @@ sub run {
     $grub_param .= ' console=hvc0'     if (get_var('ARCH') eq 'ppc64le');
     $grub_param .= ' console=ttysclp0' if (get_var('ARCH') eq 's390x');
     if (defined $grub_param) {
-        add_grub_cmdline_settings($grub_param);
+        add_grub_cmdline_settings($grub_param, update_grub => 1);
     }
```

Comment:
`update_grub => 1` good catch. It was off from time when `add_custom_grub_entries` was always run. Maybe we could optimize `update_grub` value according to whether `add_custom_grub_entries` is run (to avoid running 
generation twice), but I'm ok to have it without this small optimization.

Reply:
`update_grub => 1` isn't there because of the `(is_sle || is_opensuse) && !is_jeos` condition. It's there because `add_custom_grub_entries()` may do nothing even when it is run.

Reply:
ok, that check would need also check whether `GRUB_PARAM` is set. But forget on it, this way is safer. Ack from my side.